setup.py

```diff
@@ -43,4 +48,13 @@
         'Topic :: Documentation',
         'Topic :: Software Development :: Documentation',
     ],
+    install_requires= [
+        'Sphinx>=1.4.1',
+    ],
+    cmdclass = {
+        'compile_catalog': babel.compile_catalog,
```

Comment:
If babel was not imported, this would be an undefined variable access. You can see this failure on Travis.

In my opinion `setup.py` modifications are not needed at all. This project is using setuptools, and babel is [installing an entry point for setuptools](http://babel.pocoo.org/en/latest/setup.html), so these commands are available automatically.